### PR TITLE
feat: improve responsiveness on Profile page

### PR DESCRIPTION
**Pull request by whamester**
Code review on #146 

- As the width of the settings panels was fixed, there was a gap between 678px and 900px (approx) that was causing a vertical scroll on the page --- This was addressed by setting min and max width to #sideMenu and .account-settings_content up to 1440px, after 1440px the width is fixed

- Avoid setting the font styles directly with variables, let's use preset classes to keep these styles in one place.

- Adding flex-wrap to .account-settings__profile-picture, this to prevent overflowing.

- Select styles were changed to get closer to the mockups, still not 100% but closer.

### CODE REVIEW

File: pages/settings/index.html [hunk 1]
```diff
@@ -1,139 +1,142 @@
...
+          </div>
 
-            <div class="account-settings__email form-field">
-              <label for="email" class="text-body-2 medium"> Email address </label>
+          <form id="profileForm">
+            <div class="form-field">
+              <label for="name" class="text-body-2 medium"> Name </label>
               <div class="form-field__input-container">
-                <input type="email" id="email" class="text-body-2 regular" disabled />
+                <input type="text" id="name" required class="text-body-2 regular" />
```
Comment (whamester) [inline pages/settings/index.html:81]: I'm not sure if these changes are just formatting or if I moved things around. I'll check this in the morning.

File: pages/settings/style.css [hunk 2]
```diff
@@ -23,18 +23,16 @@ h1 {
...
   align-items: center;
   gap: 0.5rem;
+  padding: 0.5rem 1rem;
   border: none;
   border-radius: 1rem;
-  font-size: var(--body-1-large-size);
-  letter-spacing: var(--body-1-large-letter-spacing);
-  line-height: var(--body-1-large-line-height);
-  font-weight: 500;
-  color: var(--neutral-neutral-900, #101928);
```
Comment (whamester) [inline pages/settings/style.css:35]: replacing color variables from
  color: var(--neutral-neutral-900, #101928);
  background: var(--secondary-secondary-200, #edf4f2);

to

 color: var(--neutral-900);
  background: var(--secondary-200);

File: pages/settings/style.css [hunk 3]
```diff
@@ -61,11 +68,7 @@ h1 {
 }
 
 option {
-  font-size: var(--body-1-large-size);
-  letter-spacing: var(--body-1-large-letter-spacing);
-  line-height: var(--body-1-large-line-height);
-  font-weight: 400;
-  color: var(--neutral-neutral-500, #667185);
+  color: var(--neutral-500, #667185);
```
Comment (whamester) [inline pages/settings/style.css:71]: same here:

 - color: var(--neutral-neutral-500, #667185);
 + color: var(--neutral-500, #667185);

**Comment by poojachauhan001** [comment]
@whamester 
 
Whole page is not visible. I can't scroll further down.
image

**Review by ReinhardtBotha — APPROVED** [review]
👌

**Comment by whamester** [comment]
> @whamester
> 
> Whole page is not visible. I can't scroll further down. image

Addressed

**Comment by poojachauhan001** [comment]
Looks good to me. 

Just 2 more things, I'd like to add on this page.

1. Make the turn off state of toggle button as in the design file.
image

2. On mobile, the green ab should have rounded corners. Design [link](https://www.figma.com/file/EFlGK158VL4Cphasesw2aS/UI-Design?type=design&node-id=595%3A19971&mode=design&t=RtKHzcpWkYGyOv21-1).
image

**Pull request merged**